### PR TITLE
ci: runtime upgrade test

### PR DESCRIPTION
**Pull request by lumir-mrkva**

### CODE REVIEW

File: scripts/upgrade-runtime/index.js [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+  await waitForBlock(3);
+
+  console.log('submitting runtime upgrade');
+  watchEventsFromSections(["sudo", "parachainSystem"]);
+  await waitTx(sudo);
+
+  await new Promise(resolve => onEvent('parachainSystem.ValidationFunctionApplied', resolve));
+  api.rpc.chain.subscribeNewHeads(async header => {
+    const newSpec = getSpecVersion();
+    if (specVersion < newSpec) {
```
Comment (jak-pan) [inline scripts/upgrade-runtime/index.js:81]: I'd try waiting here for a few blocks to check if the chain is running after upgrade

Comment (jak-pan) [reply scripts/upgrade-runtime/index.js:81]: Maybe do some tx?

Comment (lumir-mrkva) [reply scripts/upgrade-runtime/index.js:81]: i thought that chain continuing to produce blocks is sufficient enough

Comment (lumir-mrkva) [reply scripts/upgrade-runtime/index.js:81]: it actually takes few block before api initialises into the new spec

Comment (lumir-mrkva) [reply scripts/upgrade-runtime/index.js:81]: added tx anyway

File: runtime/basilisk/src/lib.rs [hunk 2]
```diff
@@ -106,7 +106,7 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
 	spec_name: create_runtime_str!("basilisk"),
 	impl_name: create_runtime_str!("basilisk"),
 	authoring_version: 1,
-	spec_version: 26,
+	spec_version: 27,
```
Comment (jak-pan) [inline runtime/basilisk/src/lib.rs:109]: We don't need to bump this since there is no change to runtime. Is there?

Comment (lumir-mrkva) [reply runtime/basilisk/src/lib.rs:109]: not really, this was changed for testing purposes

**Review by jak-pan — COMMENTED** [review]
LGTM

**Review by jak-pan — APPROVED** [review]

**Pull request merged**